platform-sdk/swirlds-common/src/testFixtures/java/com/swirlds/common/test/fixtures/merkle/util/MerkleTestUtils.java

```diff
@@ -990,13 +991,28 @@ public static <T extends MerkleNode> T testSynchronization(
             final TeachingSynchronizer teacher;
 
             if (latencyMilliseconds == 0) {
-                learner = new LearningSynchronizer(
-                        getStaticThreadManager(),
-                        streams.getLearnerInput(),
-                        streams.getLearnerOutput(),
-                        startingTree,
-                        streams::disconnect,
-                        reconnectConfig);
+                learner =
+                        new LearningSynchronizer(
+                                getStaticThreadManager(),
+                                streams.getLearnerInput(),
+                                streams.getLearnerOutput(),
+                                startingTree,
+                                streams::disconnect,
+                                reconnectConfig) {
+
+                            @Override
+                            protected StandardWorkGroup createStandardWorkGroup(
+                                    ThreadManager threadManager,
+                                    Runnable breakConnection,
+                                    Function<Throwable, Boolean> reconnectExceptionListener) {
+                                return new StandardWorkGroup(
+                                        threadManager,
+                                        "test-learning-synchronizer",
+                                        breakConnection,
+                                        reconnectExceptionListener,
+                                        true);
+                            }
+                        };
                 final PlatformContext platformContext =
                         TestPlatformContextBuilder.create().build();
                 teacher = new TeachingSynchronizer(
```

Comment:
Why isn't the teaching synchronizer covered?

Reply:
Ah, missed it. Thank you. I redefined `createStandardWorkGroup` here as well.